apps/journeys-admin-e2e/README.md

```
@@ -0,0 +1,189 @@
+| `DEPLOYMENT_URL`           | Fallback URL (e.g. a Vercel preview URL)           |
+| `PLAYWRIGHT_EMAIL`         | Email of the pre-existing admin test account       |
+| `PLAYWRIGHT_PASSWORD`      | Password for the admin test account                |
+| `PLAYWRIGHT_USER`          | Display name for the admin test account            |
+| `PLAYWRIGHT_TEAM_NAME`     | Team name used by the admin test account           |
+| `EXAMPLE_EMAIL_TOKEN`      | Static OTP token used during new user registration |
+
+If none of the URL variables are set, tests fall back to `http://localhost:4200`.
+
+## Project Structure
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)